src/style.css

```diff
@@ -36,10 +38,17 @@ input {
 }
 
 button {
-  box-shadow: 3px 4px;
+  box-shadow: 1px 2px;
+  float: right;
+  border-radius: 5px;
+  cursor: pointer;
 }
 
-button[type='submit'] {
-  float: right;
-  margin-right: 19%;
+button:hover {
+  background-color: blue;
+  color: white;
+}
+
+form {
+  width: 185px;
```

Comment:
- Kindly, your refresh button, input field and submit buttons are not followed the wireframe, please follow the wireframe to have better design👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/217761089-ec0e75ed-545f-40db-8719-3340f9b59c9e.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/217761104-41811757-4b69-4cbb-b154-38b16fa42d35.png" width="600">|


😎️
🤝️

Reply:
Hello @binhussen 
Thanks for the feedback however I think you did not look at the latest changes because, from the screenshot you sha
![new](https://user-images.githubusercontent.com/58359456/217767787-d8aab73d-95d9-4747-a57d-9102482fbbb8.png)
red, the output is totally different from what I have

Reply:
yes @Zeph180 you are right, sorry for that  you can request another review it will be approved